### DISCUSSION TITLE
Somethings Wrong with My Running

### DISCUSSION THREAD

**Discussion by bobomie** [nzahasan/Tank-Model#19] [General]
When I attempt to run the command request "tank_cmd.py new-project project_name  ", there is an ERROR : Unable to move the Cache, may I ask you for any possible help? I would be definately  appreciate it if possible

**Comment by nzahasan** [comment]
@bobomie need the see the full error log as text, operating os, and installed python version.

**Reply by bobomie** [reply]
thank you so much！I found  the problem maybe on the installing of Anaconda，now i am trying others tool
